sampleapp/src/main/java/com/dailymotion/android/player/sampleapp/SampleActivity.kt

```diff
@@ -32,13 +32,22 @@ import java.util.concurrent.TimeUnit
 class SampleActivity : AppCompatActivity(), View.OnClickListener {
 
     companion object {
-        const val DEFAULT_VIDEO_ID = "x70val9"
+
+        const val DEFAULT_VERTICAL_VIDEO_ID = "x7h2j4q"
+        const val DEFAULT_HORIZONTAL_VIDEO_ID = "x70val9"
         const val DEFAULT_PLAYLIST_ID = "x5zhzj"
+        const val DEFAULT_SCALE_MODE = "fit"
         const val DEFAULT_QUALITY = "240"
         const val DEFAULT_VOLUME_VALUE = "1"
         const val DEFAULT_SEEK_VALUE_SEC = "30"
     }
 
+    private var scaleModeList = listOf(
+            "fit",
+            "fill"
```

Comment:
Maybe make that some constants ? `PlayerWebView.SCALE_MODE_FIT`, `PlayerWebView.SCALE_MODE_FILL`

Reply:
Done!